### DISCUSSION TITLE
Connection problems with WSL2 and Windows 11

### DISCUSSION THREAD

**Discussion by thomasbnt** [microsoft/WSL#11298] [Q&A]
**TLDR**: I **can't connect to my local network from my WSL2** (Ubuntu) machine, however **I can access the Internet**. 

## A little background before explaining my error in detail: 

I had already installed WSL2 on my Windows 10, I didn't think to back up my configurations, but everything was working perfectly.

Having had problems with the upgrade, from Windows 10 to Windows 11, I preferred to reinstall the OS from a bootable key. Everything went smoothly, no errors. Then I installed my tools and software like anyone else, including my WSL2 running Ubuntu 22 04 (the version isn't that important).

I work with remote servers via VPN, so I can connect to them locally and securely. That's where it gets stuck.

## The error

When I try to connect to a local IP **under VPN or not** via WSL, e.g. _192.168.1.25_, I can't do it. I get a Timeout message. However, when I try to connect via SSH with the classic Windows terminal, it works. 

I've tried several configurations, such as .wslconfig, or modifying `/etc/resolve.conf` in my WSL, or `/etc/wsl.conf`, and even deleting the WSL machine and starting from scratch. Reset my network on my PC. But there's nothing I can do.

## The configurations

My IP is static, I've always set it to static. That way, I know what my IP is locally. I've tested everything in terms of configuration, so my aim is to have Internet access on both my classic Windows and my WSL2, and to have access to local IPs on both machines, without there being any conflict.

**.wslconfig** (found in %UserProfile%) ([See documentation](https://learn.microsoft.com/en-us/windows/wsl/wsl-config#experimental-settings))

```
[wsl2]
memory=10GB
processors=4

[experimental]
networkingMode=mirrored
firewall=false
dnsTunneling=true
``` 

**/etc/resolv.conf**

```
nameserver 1.1.1.1
``` 

(If I don't set this configuration, it won't find the Internet).

**/etc/wsl.conf**

```
[network] 
generateResolvConf = false 
[automount] 
enabled = true 
options = "metadata" 
mountFsTab = false
```
